types/utils.go

```diff
@@ -1,18 +1,26 @@
 package types
 
 import (
+	"errors"
+	"fmt"
 	"math/rand"
 	"time"
 
 	"github.com/cometbft/cometbft/crypto/ed25519"
+	"github.com/cometbft/cometbft/p2p"
 	cmtypes "github.com/cometbft/cometbft/types"
+	"github.com/libp2p/go-libp2p/core/crypto"
 )
 
 // TestChainID is a constant used for testing purposes. It represents a mock chain ID.
 const TestChainID = "test"
 
-// GetRandomValidatorSet returns the validator set from
-// GetRandomValidatorSetWithPrivKey without the private key.
+var (
+	errNilKey             = errors.New("key can't be nil")
+	errUnsupportedKeyType = errors.New("unsupported key type")
+)
+
+// GetRandomValidatorSet returns a validator set with a single validator
 func GetRandomValidatorSet() *cmtypes.ValidatorSet {
```

Comment:
`types/utils.go` seems like a wrong place if `GetRandomValidatorSet` and `GetRandomValidatorSetWithPrivKey` are used for testing purpose?

Reply:
I agree, currently, we don't have a separate file for testing-related utils and non-testing utils though. Maybe we can separate it out in the future. Created #1345